webpack.config.js

```diff
@@ -102,20 +108,20 @@ jsLoader = {
 
 if ( CALYPSO_ENV === 'development' ) {
 	webpackConfig.plugins.push( new PragmaCheckPlugin() );
-	webpackConfig.plugins.push( new webpack.HotModuleReplacementPlugin() );
+	//webpackConfig.plugins.push( new webpack.HotModuleReplacementPlugin() );
```

Comment:
had a thought that hot mod replacement might be slowing things down. didn't really help, or fix parenting